src/sections/blog-post.liquid

```diff
@@ -0,0 +1,184 @@
+<div class="page-width">
+  {% if section.settings.title != blank %}
+    <header class="section-header text-center">
```

Comment:
Might not need this wrapper `<header>` container

Reply:
The `header` would need an associated "section-ing element", like `<body>`, `<section>`, `<article>`.

There's a suggestion to add one above, but I still agree with your comment.  This would be fine with the `<h2>` as there isn't much going on in this `<header>`.

useful source: https://internetingishard.com/html-and-css/semantic-html/#headers

Reply:
@tauthomas01 something to be conscious of: Starter theme should be shipping with classes that are unused.  In this case `section-header`.  The `text-center` class is used in the base CSS, but I feel that should be removed as well.  No need to force a centered heading on folks as a starting point.

Reply:
Yes, I removed `<header>` since there are not much information to show.